config/routes.rb

```diff
@@ -13,4 +13,5 @@
   
   get 'cart', to: 'carts#show'
   post 'add_product/:id', to: 'carts#add_product', as: 'add_product'
+  delete 'delete_products', to: 'carts#delete_products'
```

Comment:
może lepiej zmienić do 
```suggestion
  delete 'cart', to: 'carts#delete'
```

Reply:
pewnie! z tą uwagą, że metoda powinna się nazywać destroy :)